File: CodeChef/JulyChallenge2020/ChefandCardGame/CRDGAME.cpp
#include <iostream>
using namespace std;

int main() {
	// your code goes here
	
    //below 2 lines are for running code on your local machine 
    //Remove these 2 lines while using code online on codechef as they provide thier own input
    freopen("input.txt", "r", stdin);
    freopen("output.txt", "w", stdout);
    
	int t;
	cin>>t;
	for(int i=0;i<t;i++)
	{
	    int n;
	    cin>>n;
	    int chef=0;
	    int monty=0;
	    for(int j=0;j<n;j++)
	    {
	        int a;
	        int b;
	        cin>>a>>b;
	        int suma=0;
	        int sumb=0;
	        while(a!=0)
	        {
	            int d=a%10;
	            a=a/10;
	            suma+=d;
	        }
	        while(b!=0)
	        {
	            int d=b%10;
	            b=b/10;
	            sumb+=d;
	        }
	        if(suma>sumb)
	        {
	            chef++;
	        }
	        else if(suma==sumb)
	        {
	            chef++;
	            monty++;
	        }
	        else
	        {
	            monty++;
	        }
	    }
	    if(monty>chef)
	        cout<<"1"<<" "<<monty<<endl;
	    else if(monty==chef)
	        cout<<"2"<<" "<<monty<<endl;
	    else
	        cout<<"0"<<" "<<chef<<endl;
	}
	
	return 0;
}

File: CodeChef/JulyChallenge2020/ChefandCardGame/CRDGAME.py
import sys 

#   below 2 lines are for running code on your local machine 
#   Remove these 2 lines while using code online on codechef as they provide thier own input
sys.stdin = open("input.txt",'r')
sys.stdout = open("output.txt",'w')

# cook your dish here
t = int(input())

for i in range(t):
    n=int(input())
    chef=0
    monty=0
    for j in range(n):
        a ,b = map(int,input().split())
        suma = 0
        sumb = 0
        
        while a!=0:
            d=a%10
            a=a//10
            suma+=d

        while b!=0:
            d=b%10
            b=b//10
            sumb+=d

        if suma>sumb:
            chef+=1
        elif suma==sumb:
            chef+=1
            monty+=1
        else:
            monty+=1
            
    if monty>chef:
        print("1",monty)
    elif monty==chef:
        print("2",monty)
    else:
        print("0",chef)
